design-system/materials/internals/TOKENS.md

```diff
@@ -17,8 +17,7 @@ The tokens give semantic meaning to our plain palette of choices..
   - [Aspect: Component Group](#aspect-component-group)
   - [Aspect: State](#aspect-state)
     - [`active` vs `focused`](#active-vs-focused)
-  - [Aspect: Modifier](#aspect-modifier)
-    - [on-dark](#on-dark)
+  - [Aspect: Variant](#aspect-variant)
```

Comment:
It seems there are a couple of images referenced in this document which contain the `on` modifier.
Perhaps we would need to create new ones.

Reply:
I haven’t even noticed that, thanks for pointing! Do you have any idea how this image was created? Maybe we could manually add some examples?

Reply:
Sorry but I don't know.
Maybe @emmenko remembers?

Reply:
No idea. We can also simply use a code block or something, I don't think we need an image necessarily.

Reply:
Okay, I'll create a block then.